scaffold/convert/search.template.js

```diff
@@ -786,6 +786,98 @@ const getList = (z, bundle) => {
     });
 <% } %>
 };
+<% }
+
+if (inputFieldFullScripting) { %>
+const getInputFields = (z, bundle) => {
+  const scripting = require('../scripting');
+  const legacyScriptingRunner = require('zapier-platform-legacy-scripting-runner')(scripting);
+
+  bundle._legacyUrl = '<%= CUSTOM_FIELDS_URL %>';
+  bundle._legacyUrl = replaceVars(bundle._legacyUrl, bundle, {});
+
+  // Do a _custom_action_fields() from scripting.
```

Comment:
This comment should mention `_custom_search_fields()`, and I noticed the same pattern below.